analytics-webapps/src/main/webapp/vue-app/js/utils.js

```diff
@@ -25,6 +25,18 @@ export function loadUser(users, userId) {
   }
 }
 
+export function getPage(siteType, siteName) {
```

Comment:
If you will return multiple pages, it's better to call method `getPages`, else, it could be better to retrieve only the requested page.